share/spice/chefio/chefio.js

```diff
@@ -0,0 +1,36 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_chefio = function(api_result){
+        if (!api_result || api_result.error) {
+            return Spice.failed('chefio');
+        }
+
+        Spice.add({
+            id: "chefio",
+            name: "software",
+            data: api_result.items,
+            meta: {
+                sourceName: "Chef.io",
+                sourceUrl: "http://chef.io"
+            },
+            normalize: function(item) {
+                var a = {
```

Comment:
Just FYI, you can actually return an annonymous object, rather than creating a named one and immediately returning one:

```js
normalize: function(item) {
    return {
        url: ...,
        title: ....,
        subtitle: ...
    }
},
...
```